src/dust.ts

```diff
@@ -0,0 +1,196 @@
+import { Random, RngState } from "@reside-ic/random";
+
+import { base } from "./base";
+import type { DustModel, DustModelInfo, DustModelConstructable } from "./model";
+import { Pars } from "./pars";
+import { Particle } from "./particle";
+import { DustState, dustState } from "./state";
+import { DustStateTime, dustStateTime } from "./state-time";
+
+/**
+ * Dust models
+ */
+export class Dust {
+    private readonly _Model: DustModelConstructable;
```

Comment:
```suggestion
    private readonly _model: DustModelConstructable;
```
I guess this would be more consistent, or are you deliberately pascal-casing constructables?

Reply:
yeah, this was deliberate based on how we did it in odin-js (e.g., https://github.com/mrc-ide/odin-js/blob/main/src/wodin.ts#L24-L32). If lower case is more idiomatic I can change it throughout